README.md

```diff
@@ -41,25 +42,33 @@ parsers, feeding them input, and retrieving their results. At runtime,
 parsing proceeds fully incrementally—and potentially highly
 concurrently—on input streams of arbitrary size. Compilation of Spicy
 parsers takes place either just-in-time at startup (through a C++
-compiler), or ahead-of-time either by creating pre-compiled shared
-libraries or simply by giving you C++ code that you can link into your
-application.
+compiler); or ahead-of-time either by creating pre-compiled shared
+libraries, or by giving you generated C++ code that you can link into
+your application.
 
-Spicy comes with a [Zeek](https://www.zeek.org) plugin that enables
-adding new protocols to Zeek without having to write any C++ code. You
+Spicy comes with a [Zeek plugin](https://github.com/zeek/spicy-plugin)
+that enables adding new protocol and file analyzers to
+[Zeek](https://www.zeek.org) without having to write any C++ code. You
 define the grammar, specify which Zeek events to generate, and Spicy
-takes care of the rest.
+takes care of the rest. There's also a [Zeek
+analyzers](https://github.com/zeek/spicy-analyzers) package that
+provides Zeek with several new, Spicy-based analyzers.
 
+See our [collection of example grammars](https://docs.zeek.org/projects/spicy/en/latest/programming/examples.html#examples)
+to get a sense of how Spicy looks like.
```

Comment:
`to get a sense of how Spicy looks like.` is too colloquial. Suggest:
`to get a sense of how Spicy looks.` or `to get a sense of what Spicy looks like.`